src/DeviceStatusPage/ExchangedBytesCard.tsx

```diff
@@ -74,16 +74,19 @@ const ExchangedBytesCard = ({ astarte, device }: ExchangedBytesCardProps): React
   let interfaceBytes = 0;
   let interfaceMessages = 0;
   fullList.forEach((interfaceStats: AstarteDeviceInterfaceStats) => {
-    interfaceBytes += interfaceStats.exchangedBytes;
-    interfaceMessages += interfaceStats.exchangedMessages;
-    computedStats.push({
-      name: `${interfaceStats.name} v${interfaceStats.major}.${interfaceStats.minor}`,
-      bytes: interfaceStats.exchangedBytes,
-      bytesPercent: interfaceBytes !== 0 ? (interfaceStats.exchangedBytes * 100) / totalBytes : 0,
-      messages: interfaceStats.exchangedMessages,
-      messagesPercent:
-        interfaceMessages !== 0 ? (interfaceStats.exchangedMessages * 100) / totalMessages : 0,
-    });
+    if (interfaceStats.exchangedMessages !== 0) {
+      interfaceBytes += interfaceStats.exchangedBytes;
+      interfaceMessages += interfaceStats.exchangedMessages;
+
+      computedStats.push({
+        name: `${interfaceStats.name} v${interfaceStats.major}.${interfaceStats.minor}`,
+        bytes: interfaceStats.exchangedBytes,
+        bytesPercent: interfaceBytes !== 0 ? (interfaceStats.exchangedBytes * 100) / totalBytes : 0,
+        messages: interfaceStats.exchangedMessages,
+        messagesPercent:
+          interfaceMessages !== 0 ? (interfaceStats.exchangedMessages * 100) / totalMessages : 0,
+      });
+    }
```

Comment:
this can be simplified by chanining a filter
```typescript
 fullList
  .filter((interfaceStats: AstarteDeviceInterfaceStats) => interfaceStats.exchangedBytes > 0)
  .forEach((interfaceStats: AstarteDeviceInterfaceStats) => {
    [...]
  });